gradle.properties

```diff
@@ -12,13 +12,13 @@ ballerinaGradlePluginVersion=1.0.0
 ballerinaLangVersion=2201.5.0
 
 #stdlib dependencies
-stdlibIoVersion=1.4.0
-stdlibLogVersion=2.7.0
-stdlibFileVersion=1.6.0
+stdlibIoVersion=1.4.1
+stdlibLogVersion=2.7.1-20230522-155500-d73bdb5
+stdlibFileVersion=1.7.1-20230523-104800-e56ed72
 
 #stdlib transitive dependencies
 stdlibOsVersion=1.6.0
 stdlibTimeVersion=2.2.4
-stdlibRegexVersion=1.4.0
+stdlibRegexVersion=1.4.3
```

Comment:
Didnt we remove the `ballerina/regex` usage here?

Reply:
Removed in [324a75c](https://github.com/ballerina-platform/module-ballerina-yaml/pull/9/commits/324a75c2e94c0f941aab4ed1c6948ea35a4155f7)